simplipy/camera.py

```diff
@@ -0,0 +1,102 @@
+from urllib.parse import urlencode
+
+from simplipy.entity import Entity
+
+MEDIA_URL_BASE: str = "https://media.simplisafe.com/v1"
+DEFAULT_VIDEO_WIDTH: int = 1280
+DEFAULT_AUDIO_ENCODING: str = "AAC"
+
+CAMERA_MODEL_CAMERA: str = "CAMERA"
+CAMERA_MODEL_DOORBELL: str = "DOORBELL"
+
+MODEL_TO_TYPE = {"SS001": CAMERA_MODEL_CAMERA, "SS002": CAMERA_MODEL_DOORBELL}
+
+
+class Camera(Entity):
+    """A SimpliCam."""
+
+    @property
+    def camera_settings(self) -> dict:
+        """Return the camera settings.
+
+        :rtype: ``dict``
+        """
+        return self.entity_data["cameraSettings"]
+
+    @property
+    def camera_type(self) -> str:
+        """Return the type of camera.
+
+        :rtype: ``str``
+        """
+        return MODEL_TO_TYPE[self.entity_data["model"]]
```

Comment:
Any concerns about a different model type (that we don't know about) popping up? If so, this will throw an unhandled exception. Take a look at how [`system/__init__.py`](https://github.com/bachya/simplisafe-python/blob/f8066d6d451ba4ea51cfe1f135cc69696a090e7e/simplipy/system/__init__.py#L261-L273) handles it (not saying you need an `Enum` if you don't want to, but you'll get the idea).